### PR TITLE
Replace references to gitter, mailing list -> forum

### LINKED ISSUE #671: MNE Forum
Hello all,

we're (MNE-Python folks) starting to switch all our user support & discussions over to our shiny new Discourse forum, https://mne.discourse.group/

It would make **total** sense to also direct MNE-BIDS usage questions there. Also I'd like to encourage all interested parties, esp. developers, to join us in the forum to help create a friendly, welcoming, but also knowledgeable space 👍 

We should also consider adding a link to the forum to our docs.

cc @adam2392 @sappelhoff @alexrockhill

### PR DESCRIPTION
**Pull request by hoechenberger**
Remove all references to Gitter and the mailing list in favor of references to the forum.

Closes #671

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-
-[![Gitter](https://badges.gitter.im/mne-tools/mne-bids.svg)](https://gitter.im/mne-tools/mne-bids)
```
Comment (sappelhoff) [inline README.md:2]: is there a discourse badge we could use?

File: README.md [hunk 2]
```diff
@@ -29,8 +28,16 @@ The documentation can be found under the following links:
 - for the [stable release](https://mne.tools/mne-bids/)
 - for the [latest (development) version](https://mne.tools/mne-bids/dev/index.html)
 
+Getting Support
+---------------
+
+For any usage questions, please post to the
+[MNE Forum](https://mne.discourse.group). Be sure to add the `mne-bids` tag to
+your question.
```
Comment (sappelhoff) [inline README.md:36]: nice :+1:

**Review by sappelhoff — APPROVED** [review]
unrelated: Richard did you store the mne-study-template poster somewhere citable? (zenodo, etc.?) If I wanted to cite the mne-study-template ... :)

File: README.md (hunk 1 above)
Comment (hoechenberger) [reply README.md:2]: I tried using https://user-images.githubusercontent.com/1681963/52239617-e2683480-289c-11e9-922b-5da55472e5b4.png but then wasn't sure it it's really helpful… I can add it again for you to get an impression of what it looks like, hang on a minute…

Comment (hoechenberger) [reply README.md:2]: @sappelhoff I tried and tried, and everything turned out not to be looking great. I don't want to spend much more time on this – if you have any good idea, go for it :) As a "compromise", for now I've added a large Discourse logo to the "Get Help" section

Comment (sappelhoff) [reply README.md:2]: yes, that looks good - thanks!

File: README.md (hunk 2 above)
Comment (hoechenberger) [reply README.md:37]: btw @sappelhoff you can subscribe to individual tags, e.g. to get notifications for posts tagged as `men-bids`, go to https://mne.discourse.group/tag/mne-bids and click the bell

**Comment by hoechenberger** [comment]
> unrelated: Richard did you store the mne-study-template poster somewhere citable? (zenodo, etc.?) If I wanted to cite the mne-study-template ... :)

Hm I'm not sure, I need to look. Meaning I probably didn't! But can do. Is it urgent?

**Comment by hoechenberger** [comment]
Also there will very soon be a study-template release, which you will be able to cite too I suppose

**Comment by hoechenberger** [comment]
Relevant CIs have passed, the fail is due to a dataset download error. Merging.

**Comment by hoechenberger** [comment]
Hum I closed it accidentally. :D

**Comment by sappelhoff** [comment]
not urgent, within 5 months suffices probably. and yes - I can also just cite the zenodo release :+1:

**Comment by hoechenberger** [comment]
> not urgent, within 5 months suffices probably. and yes - I can also just cite the zenodo release 👍

In that case – just ping me again in 4 months or so? ;)

**Pull request merged**